snackager/src/__integration-tests__/__snapshots__/bundler.test.ts.snap

```diff
@@ -200,6 +200,61 @@ Object {
 }
 `;
 
+exports[`bundler creates bundle for graphql@14.0.0 1`] = `
+Object {
+  "files": Object {
+    "android": Object {
+      "bundle.js": Object {
+        "externals": Array [
+          "./graphql",
+          "./type",
+          "./language",
+          "./execution",
+          "./subscription",
+          "./validation",
+          "./error",
+          "./utilities",
+        ],
+        "size": 20124,
+      },
+    },
+    "ios": Object {
+      "bundle.js": Object {
+        "externals": Array [
+          "./graphql",
+          "./type",
+          "./language",
+          "./execution",
+          "./subscription",
+          "./validation",
+          "./error",
+          "./utilities",
+        ],
+        "size": 20120,
+      },
+    },
+    "web": Object {
+      "bundle.js": Object {
+        "externals": Array [
```

Comment:
wait, I don't think these externals are right. Shouldn't they be built into the package?